index.html

```diff
@@ -50,7 +50,8 @@
 </head>
 
 <body>
-  <section id='abstract' data-include="spec/01-abstract.md" data-include-format='markdown'></section>
+  <section id='abstract' data-include="spec/00-abstract.md" data-include-format='markdown'></section>
+  <section id='abstract' data-include="spec/01-processing-model.md" data-include-format='markdown'></section>  
```

Comment:
Also current master doesn't have `01-processing-model.md` file. Looks like some merge problem

Reply:
renaming in this PR: `spec/01-abstract.md → spec/01-problem-statement.md`, not to processing model

Reply:
this was an issue with visual studio not updating local files. Will fix